docs/pages/versions/unversioned/sdk/permissions.md

```diff
@@ -6,16 +6,42 @@ sourceCodeUrl: 'https://github.com/expo/expo/tree/sdk-36/packages/expo-permissio
+
+> **Note:** apps using permissions without descriptions _may be rejected from the App Store_. (see the [App Store Deployment Guide](../../distribution/app-stores/#system-permissions-dialogs-on-ios))
+
+### Permissions on Android
+
+To use permissions on Android, they have to be defined in your `AndroidManifest.xml` first. Some Expo and React Native modules imply these permissions by default. If you use `expo-location`, for example, both the `ACCESS_COARSE_LOCATION` and `ACCESS_FINE_LOCATION` are implied and added to your `AndroidManifest.xml` when building a standalone app.
+
+You can limit these permissions by defining `android.permissions` property in your [`app.json` manifest](../workflow/configuration/#android) in the managed workflow. Only the minimum required permissions for Expo and the permissions listed in the `android.permissions` array are added. When using the bare workflow, you have to [blacklist permissions in your `AndroidManifest.xml`](#excluding-android-permissions-in-bare-workflow) manually.
+
+See the [`Permission types`](#permission-types) below to learn about which Android permissions are added. You can find a full list of all available permissions in the [Android Manifest.permissions reference](https://developer.android.com/reference/android/Manifest.permission).
+
+> **Note:** apps using dangerous or signature permissions without valid reasons _may be rejected by Google_. Make sure you follow the [Android permissions best practices](https://developer.android.com/training/permissions/usage-notes) when submitting your app.
+
+> **Note:** by default, the permissions implied by the modules you installed are added to the `AndroidManifest.xml`. To exclude permissions, you have to define the `android.permissions` manifest property or [blacklist them in the bare workflow](#excluding-android-permissions-in-bare-workflow).
+
+## Usage
+
+### Manually testing permissions
+
+Often you want to be able to test what happens when you reject a permission to ensure that it has the desired behavior. An operating-system level restriction on both iOS and Android prohibits an app from asking for the same permission more than once (you can imagine how this could be annoying for the user to be repeatedly prompted for permissions). So in order to test different flows involving permissions, you may need to uninstall and reinstall the Expo app. In the simulator this is as easy as deleting the app and expo-cli will automatically install it again next time you launch the project from it.
+
 ### Usage in bare workflow
```

Comment:
- I think this is only required for the iOS permissions, right? The doc doesn't make that clear, but I can request the permissions on Android without adding any of these libraries.
- Needs to be double-checked and updated if necessary

Reply:
I double-checked this and added the `on iOS` mention.